package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pinterest/graphql-lint-rules",
-  "version": "0.3.0",
+  "version": "0.4.5",
```

Comment:
I think you still want `0.3.0` here.

Reply:
Yeah, I was changing that at the time you were reviewing this, I think.